components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java

```diff
@@ -13933,6 +13935,7 @@ public void deleteEnvironment(String uuid) throws APIManagementException {
             connection.setAutoCommit(false);
             try (PreparedStatement prepStmt = connection.prepareStatement(SQLConstants.DELETE_ENVIRONMENT_SQL)) {
                 prepStmt.setString(1, uuid);
+                ChoreoApiMgtDAO.getInstance().deleteEnvToDataPlaneMapping(connection, uuid);
```

Comment:
should we move this after `connection.setAutoCommit(false);` and before `try (PreparedStatement prepStmt.. ` to make it easy to read?

Reply:
Fixed